internal/trace/trace.go

```diff
@@ -32,16 +32,15 @@ import (
 )
 
 const (
-	telemetryPlatformTracingOpenCensus    = "opencensus"
-	telemetryPlatformTracingOpenTelemetry = "opentelemetry"
-	telemetryPlatformTracingVar           = "GOOGLE_API_GO_EXPERIMENTAL_TELEMETRY_PLATFORM_TRACING"
+	TelemetryPlatformTracingOpenCensus    = "opencensus"
+	TelemetryPlatformTracingOpenTelemetry = "opentelemetry"
+	TelemetryPlatformTracingVar           = "GOOGLE_API_GO_EXPERIMENTAL_TELEMETRY_PLATFORM_TRACING"
```

Comment:
same for ENV. Please let me know if i am missing something.

Reply:
cannot assign to `TelemetryPlatformTracingVar` because it is a `const`